studies/queries.py

```diff
@@ -145,7 +145,7 @@ def get_responses_with_current_rulings_and_videos(study_id, preview_only):
         ).recording_method_is_pipe
         videos_per_response[video["response_id"]].append(
             {
-                "aws_url": get_download_url(video["full_name"], recording_is_pipe),
+                "aws_url": get_url(video["full_name"], recording_is_pipe, False),
```

Comment:
The comment preceding this change might need to go.

Reply:
@okaycj you mean I should remove this comment?

> \# Not prefetching videos above because the inability to use values() and
    # prefetch_related in tandem without combinatorial explosion of result set is precluding us
    # from relying on django's join machinery. Instead, we need to manually join here.
    # See: https://code.djangoproject.com/ticket/26565n
    
Does that comment no longer apply? 

I don't think I did anything to change this. This is the query that gets all of the consent videos for the consent manager view, so I just updated the function name and used `False` for the attachment header param, since this view plays the videos in the browser. So it should work the same as before.

Reply:
I don't really care, If you feel more comfortable leaving it, we can.  It's just that I don't think it applies to the code it's referencing and the Django ticket link doesn't work.